Runtimes/Supplemental/CMakeLists.txt

```
@@ -22,6 +22,7 @@ endif()
 
 set(COMMON_OPTIONS
   -DBUILD_SHARED_LIBS=${BUILD_SHARED_LIBS}
+  -DSwift_SDKROOT=${SDKROOT}
```

I _believe_ the name of the source variable needs to be amended

```suggestion
  -DSwift_SDKROOT=${Swift_SDKROOT}
```